src/scrape_up/pinterest/pinterest.py

```diff
@@ -0,0 +1,63 @@
+from bs4 import BeautifulSoup as bs 
+import requests
+
+class Pinterest():
+
+    """
+        Create an instance of `Flipkart` class.
+        ```python
+        pin = Pinterest()
+        ```
+        | Methods                | Details                                                            |
+        | ---------------------- | ------------------------------------------------------------------ |
+        | `.get_today()`         | Returns the list of todays topics                                  |
+        | `.get_photo(your url)` | Returns the link to the image (so you dont need an account)        |
+    """
+    def __init__(self):
+        pass
+
+    def get_today(self):
+        """
+            Class - `Pinterest`
```

Comment:
Format docstring, check modules like GitHub and IMDB and you'll get an idea about how we format docstring of each mehods.